nuxeo-core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/api/CoreSession.java

```diff
@@ -102,6 +102,18 @@
     String ALLOW_VERSION_WRITE = "allowVersionWrite";
 
     /**
+     * A context data key which represents the source of current session calls. For example:
+     * <ul>
+     *     <li>fileimporter-NoteImporter</li>
+     *     <li>drive</li>
+     *     <li>rest</li>
```

Comment:
Are these example right, given that the code in `JSONDocumentObject` does a `toUpperCase`?